src/app/app.less

```diff
@@ -15,19 +15,6 @@ a {
   float: right;
 }
 
-#betaStamp {
-  position: absolute;
-  top: 10px;
-  @media(max-width:@navbar-small-breakpoint-max) {
-    left: 215px;
-  }
-  @media(min-width:@navbar-small-breakpoint) {
-    left: 285px;
-  }
-  color: @lightGray;
-  font-size: 16px;
-  .rotate(-15deg);
-}
```

Comment:
BloomLibrary doesn't need the css files checked in.  Grunt compiles all the less into BloomLibrary-0.1.0.css